test/test_helper.rb

```diff
@@ -16,6 +16,17 @@ def using_mysql?
 end
 
 require File.expand_path("../dummy/config/environment.rb",  __FILE__)
+if Gem::Version.new(ActiveRecord::VERSION::STRING) >= Gem::Version.new('5.0.0.beta1')
+  ActiveSupport.on_load(:action_controller) do
+    ActionController::TestCase.send(:include, Rails::Controller::Testing::TestProcess)
+    ActionController::TestCase.send(:include, Rails::Controller::Testing::TemplateAssertions)
+
+    ActionDispatch::IntegrationTest.send(:include, Rails::Controller::Testing::TemplateAssertions)
+    ActionDispatch::IntegrationTest.send(:include, Rails::Controller::Testing::Integration)
+
+    ActionView::TestCase.send(:include, Rails::Controller::Testing::TemplateAssertions)
+  end
```

Comment:
This whole `on_load` could use a comment, e.g. why are we including these modules, why are we using `send`, why `on_load`, etc. I don't personally understand any of it, so the comment is mostly for me :p